components/tabs/stories/tabs-horizontal.stories.js

```diff
@@ -30,11 +30,59 @@ export default {
   }
 };
 
-export const Default = Template.bind({});
-Default.args = horizontalProps;
+const horiztonalTabsGroup = ({
+	...args
+  }) => {
+    return html `
+      <div style="display: grid; gap: 1rem;" >
+        ${Typography({
+          semantics: "heading",
+          size: "s",
+          content: ["Horizontal default"],
+        })}
+        ${TabsGroup({...args, })}
+        ${Typography({
+          semantics: "heading",
+          size: "s",
+          content: ["Horizontal emphasized"],
+        })}
+        ${TabsGroup({...args, isEmphasized: true})}
+        ${Typography({
+          semantics: "heading",
+          size: "s",
+          content: ["Horizontal quiet"],
+        })}
+        ${TabsGroup({...args, isQuiet: true})}
+        ${Typography({
+          semantics: "heading",
+          size: "s",
+          content: ["Horizontal quiet compact"],
+        })}
+        ${TabsGroup({...args, isQuiet: true, isCompact: true})}
+      </div>
+    `
+};
+
+const overflowTabsGroup = ({
+  ...args
+}) => {
+  return html `
```

Comment:
I can't really tell if this is working in the snapshots or not - can you add some more contextual data to the snapshot so we can tell at a glance if it's handling overflow correctly?